src/components/navbar/Navbar.css

```diff
@@ -0,0 +1,66 @@
+.navbar__wrapper {
+  text-align: center;
+  width: 100%;
+  background-color: #333333;
+  height: 5vh;
+  color: #d7d7d7;
+  transition: all 0.2s ease-in;
+  z-index: 10;
+}
+.navbar__Link {
+  color: #d7d7d7;
+  text-decoration: none;
+}
+.Navbar__logo {
+  color: #d7d7d7;
+  text-decoration: none;
+}
+.Navbar__logo--scrolled {
+  color: #333333;
+}
+.navbar__wrapper--scrolled {
+  position: fixed;
+  top: 0;
+  backdrop-filter: blur(100px) contrast(0.8);
+  background-color: transparent;
+  color: #333333;
+}
+
+.navbar__Link--scrolled {
+  color: #333333;
+}
+.navbar__wrapper ul {
+  list-style: none;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 90%;
+  margin: auto;
+  height: 100%;
+}
+
+.navbar__linkWrapper > ul {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+}
+
+ul > a > li {
+  margin: 0 16px;
+  transition: all 0.1s ease-in;
+  cursor: pointer;
+}
+
```

Comment:
targetting using classes will be better overall

Reply:
Thank you so much bro for the review will make changes tomorrow, I was travelling today.